init.go

```diff
@@ -156,7 +156,11 @@ func postInitPlugins(event apid.Event) {
 
 		tokenManager = createTokenManager()
 
-		go bootstrap()
```

Comment:
Where is the new bootstrap() function definition. Are all the changes in the code review?